ui/src/app/applications/components/utils.tsx

```diff
@@ -330,11 +334,11 @@ export const deletePopup = async (ctx: ContextApis, resource: ResourceTreeNode,
         api => (
             <div>
                 <p>
-                    Are you sure you want to delete {resource.kind} '{resource.name}'?
+                    Are you sure you want to delete {resource.kind} <kbd>{resource.name}</kbd>?
                 </p>
                 {isManaged ? (
                     <div className='argo-form-row'>
-                        <FormField label={`Please type '${resource.name}' to confirm the deletion of the resource`} formApi={api} field='resourceName' component={Text} />
+                        <FormField label={`Please type <kbd>${resource.name}</kbd> to confirm the deletion of the resource`} formApi={api} field='resourceName' component={Text} />
```

Comment:
The HTML tag should not be displayed in the dialog box, not sure if this is intentional
![Screen Shot 2022-08-29 at 12 06 19 PM](https://user-images.githubusercontent.com/11219262/187278752-82aa9087-21d0-46e0-a6bf-6503f27e8328.png)